tests/foreman/ui/test_adusergroup.py

```diff
@@ -599,72 +242,3 @@ def test_positive_add_external_user(self):
 
         :CaseLevel: Integration
         """
-
-    @stubbed()
-    @tier2
-    def test_positive_add_admin_role_with_org_loc(self):
-        """Associate Admin role to User Group with org and loc set.
-        [belonging to external AD User Group.]
-
-        :id: 00841778-f89e-4445-a6c6-f1470b6da32e
-
-        :setup: LDAP Auth Source should be created with Org and Location
-                Associated.
-
-        :Steps:
-            1. Create an UserGroup.
-            2. Assign admin role to UserGroup.
-            3. Create and associate an External AD UserGroup.
-
-        :expectedresults: Whether a User belonging to User Group is able to
-            access some of the pages, with the associated org and loc
-            in LDAP Auth source page as the context set.
-
-        :CaseImportance: Critical
-        """
-
-    @stubbed()
-    @tier2
-    def test_positive_add_foreman_role_with_org_loc(self):
```

Comment:
removed. as of 6.5, specifying org and loc is required, so existing tests cover this scenario.

Reply:
yes, and it is expected to perform operation defined by roles exactly in the context of location and organization specified in auth source which is a different coverage that you provided above, but very slight one

Reply:
implemented extra test as per discussion (but katello one, foreman one doesn't really make sense as foreman entities don't usually offer org/loc selection and don't tie up to them)

Reply:
and since when host group is a katello entity?

Reply:
host entity also provides selection of org and loc, but it is definitely not a katello entity.
probably we need to use whether katello entity or proper words in out statements

Reply:
I've mixed up katello and foreman, my bad. It's actually the opposite - only foreman entities have org/loc tabs, katello entities don't have anything similar. I'll update test header to show it's foreman test, thx for catching.
Regarding katello test - there's no such easy way to verify assigned org/loc. The only idea i had was to indirectly check that by switching org/loc and making sure entity isn't present in different org/loc combination, but you were against that. So unless you've changed your mind i'll probably delete that stub.

Reply:
if it is a stub and you do not provide coverage for it - then leave it as it is, but close issue as completed for porting effort. Implementation of stubs is only a good will, not a requirement for current effort